### PR TITLE
Free erica 🧦

### PR DESCRIPTION
**Pull request by nfelger**
# Short Description
- This frees Erica into a new repository at: https://github.com/digitalservice4germany/erica
- Goal is to make deployments independent from steuerlotse

# Changes
- Removes everything related to Erica from this repo
- Adapts CI/CD workflows to no longer involve Erica

# Feedback
- Anything we missed?
- Any problems you anticipate?

# Attention
This removes all binaries from the .gitignore, thus you might wanna double check that you're not adding them when working on this branch and once it is merged. :)

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

**Comment by nfelger** [comment]
(cc @punknoir101 @VictorDelCampo so you know what's going on.)

Please note the changed .gitignore file which means you are at risk of committing sensitive data (ERiC binaries and certificates). Best to make a checkout of https://github.com/digitalservice4germany/erica and to move them there once (or even before) this is merged.

File: docker-compose.ci.yml [hunk 1]
```diff
@@ -15,15 +15,3 @@ services:
...
-    image: steuerlotse_erica:${DOCKER_TAG:-latest}
-    build:
-      context: ./erica_app
-      args:
-        bucket_name: ${ERICA_BUCKET_NAME}
-        access_key_id: ${ACCESS_KEY_ID}
-        access_key: ${ACCESS_KEY}
-        endpoint_url: ${ENDPOINT_URL}
-        elster_datenlieferant: ${ELSTER_DATENLIEFERANT}
-        elster_hersteller_id: ${ELSTER_HERSTELLER_ID}
```
Comment (JulianRoesner) [inline docker-compose.ci.yml:29]: #635 would enable us to run docker compose locally to get the services running. We probably need to change/move that, too.

**Review by JulianRoesner — APPROVED** [review]
So nice 💪 The only problem I see is the question what we are going to do with docker-compose

File: .github/workflows/cicd-pipeline.yml [hunk 2]
```diff
@@ -227,14 +156,10 @@ jobs:
...
         -e DOCKER_REGISTRY='${{ env.DOCKER_REGISTRY }}' \
-        -e REGISTRY_PWD='${{ secrets.REGISTRY_PWD }}' \
-        -e CERT_PIN='${{ secrets.CERT_PIN_STAGING }}' \
         -e KUBECONFIG_BASE64='${{ secrets.CI_KUBECONFIG_B64 }}' \
-        -e ERICA_HOST='${{ env.ERICA_HOST }}' \
-        -e ERICA_SSH_KEY='${{ secrets.CI_ERICA_SSH_KEY }}' \
         -e IMAGE_TAG='${{ env.IMAGE_TAG }}' \
         -e NAMESPACE='${{ env.NAMESPACE }}' \
-        ${{ env.DOCKER_REGISTRY }}/steuerlotse_deployment
+        ${{ env.DOCKER_REGISTRY }}/steuerlotse_deployment steuerlotse
```
[the remark this answers is not in the document]
Comment (JulianRoesner) [reply .github/workflows/cicd-pipeline.yml:162]: Maybe we could add a comment to that part?

File: docker-compose.ci.yml (hunk 1 above)
Comment (nfelger) [reply docker-compose.ci.yml:29]: I commented on that PR and Jördis knows what's coming :)

File: .github/workflows/cicd-pipeline.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (nfelger) [reply .github/workflows/cicd-pipeline.yml:162]: This is the target introduced here: https://github.com/digitalservice4germany/steuerlotse-infra/pull/31
Help me understand: what is the comment that you have in mind and where should it go?

[the remark this answers is not in the document]
Comment (JulianRoesner) [reply .github/workflows/cicd-pipeline.yml:162]: I wou
```suggestion
        # The deployment image offers different targets. Use the one for steuerlotse here
        ${{ env.DOCKER_REGISTRY }}/steuerlotse_deployment steuerlotse
```

**Pull request merged**